app/src/main/res/layout/dialog_authentication.xml

```diff
@@ -17,6 +17,7 @@
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
             android:hint="@string/username"
+            android:inputType="text"
```

Comment:
I dont think this is related to the Manual Setup View, this screen is used in the webview when a user hits an authentication dialog which is after being logged in.